benchmarks/src/jmh/resources/vm.properties

```diff
@@ -1,6 +1,9 @@
 vsphere.url=https://10.173.65.98/sdk
+vsphere.host=10.172.208.208
```

Comment:
This shouldn't be public.

Reply:
I don't see any security issues, it is just an ip address like 10.10.10.10. 
Do you believe it is critical?